rules/alerting_test.go

```diff
@@ -1025,3 +1027,22 @@ func TestAlertingRule_SetNoDependencyRules(t *testing.T) {
 	rule.SetNoDependencyRules(true)
 	require.True(t, rule.NoDependencyRules())
 }
+
+func TestAlertingRule_ActiveAlertsCount(t *testing.T) {
+	rule := NewAlertingRule(
+		"TestRule",
+		nil,
+		time.Minute,
+		0,
+		labels.FromStrings("severity", "critical"),
+		labels.EmptyLabels(), labels.EmptyLabels(), "", true, nil,
+	)
+
```

Comment:
I would add `require.Equal(t, 0, rule.ActiveAlertsCount())` here.

Reply:
Done in https://github.com/prometheus/prometheus/pull/13980/commits/151f6e0ed6e9e60c0c667da294c099bc3ece8047